documentation/modules/ROOT/pages/connectors/vitess.adoc

```diff
@@ -35,15 +35,15 @@ The connector gives you the flexibility to choose to subscribe to the `MASTER` n
 
 The connector produces a change event for every row-level insert, update, and delete operation that was captured and sends change event records for each table in a separate Kafka topic. Client applications read the Kafka topics that correspond to the database tables of interest, and can react to every row-level event they receive from those topics.
 
-The underlying MySQL implementation in Vitess purges binary logs based on some configurable period of time. 
+The underlying MySQL implementation in Vitess purges binary logs based on some configurable period of time.
 Because the contents of the binlog might not be complete, the connector requires another mechanism to ensure that it captures the complete content of a particular database.
-Thus, when the connector connects to a database for the first time, it performs a consistent snapshot of the database. 
-After the connector completes the snapshot, it continues streaming changes from the exact point at which the snapshot was made. 
+Thus, when the connector connects to a database for the first time, it performs a consistent snapshot of the database.
+After the connector completes the snapshot, it continues streaming changes from the exact point at which the snapshot was made.
 In this way, the connector starts with a consistent view of all of the data, and does not omit any changes that were made while the snapshot was being taken.
 
-The connector is tolerant of failures. 
-As the connector reads changes and produces events, it records the VGTID position for each event. 
-If the connector stops for any reason (including communication failures, network problems, or crashes), after the connectors restarts, it continues reading the WAL from the point at which it stopped. 
+The connector is tolerant of failures.
+As the connector reads changes and produces events, it records the VGTID position for each event.
+If the connector stops for any reason (including communication failures, network problems, or crashes), after the connectors restarts, it continues reading the WAL from the point at which it stopped.
```

Comment:
> More accurate is it starts reading from VStream where it left off

I made an adjustment, but I wonder, would it be more accurate to say that the connector continues reading from the last VGTID that it read.

Because we do not productize the Vitess connector, I haven't done much with this documentation, but reading the doc, and knowing how things work for the other connectors, that seems to be a correct portrayal.

Reply:
Yes for "where it left off" that is represented by VGTID. I'll update

Reply:
Cool. Apologies again for failing to notice earlier that I hadn't submitted my review.

Reply:
It is actually the last VGTID that it stored (since upon recovery it reads from what it last stored), not necessarily the last one that it read.

Reply:
Edited my comment to reflect this.

> It is actually the last VGTID that it stored (since upon recovery it reads from what it last stored), not necessarily the last one that it read.